pyuploadcare/api/client.py

```diff
@@ -155,30 +246,22 @@ def _perform_request(  # noqa: C901
             headers=headers,
             cookies=cookies,
             auth=auth,
-            allow_redirects=allow_redirects,
             timeout=timeout,
         )
 
-        logger.debug(
-            f"got: status_code: {response.status_code}; "
-            f"content: {str(response.content)}; headers: {response.headers}"
-        )
-
-        if response.status_code in (401, 403):
-            raise AuthenticationError(response.content.decode())
-
-        if response.status_code in (400, 404):
-            raise InvalidRequestError(response.content.decode())
-
-        if response.status_code == 429:
-            raise ThrottledRequestError(response)
+        if PY36:
+            kwargs["allow_redirects"] = redirecting
+        elif PY37_AND_HIGHER:
+            kwargs["follow_redirects"] = redirecting
+        else:
+            raise ValueError(
+                "Unexpected set of Python version. Check the setup"
+            )
```

Comment:
this never happens, due to definition of `PY36` and `PY37_AND_HIGHER`

Reply:
Manipulation with both global flags `PY36` and `PY37_AND_HIGHER` in tests while working around different versions response potentially may lead to inproper combination of their values - I decided to prevent such behavior with explicit check of never-happening `else-`branch

Reply:
ok